devenv/docker/blocks/prometheus/docker-compose.yaml

```
@@ -4,6 +4,12 @@
       - "9090:9090"
     extra_hosts:
       - "host.docker.internal:host-gateway"
+    command: >
+      --enable-feature=remote-write-receiver
```

I think this one is deprecated according to https://prometheus.io/docs/prometheus/latest/feature_flags/#remote-write-receiver

Could we change it to?

```suggestion
     --web.enable-remote-write-receiver
```

---

@ifrost i noticed that as well, but it didn't work for me locally

---

Hmm... seems to work on my machine :D I'll check it again

---

@toddtreece I was able to add remote write target and record queries with `--web.enable-remote-write-receiver` 🤔 Are you getting any errors?

---

@ifrost i don't get errors when running `make devenv sources=prometheus`, but prometheus doesn't actually start for me.

here's the error i get:
```
$ make devenv sources=prometheus && docker logs devenv_prometheus_1
error parsing commandline arguments: unknown long flag '--web.enable-remote-write-receiver'
prometheus: error: unknown long flag '--web.enable-remote-write-receiver'
```

---

Hmm... fair enough 🤔 Not a big deal, we will get an error once it's removed anyway 👍